## The python-flask problem (oserror: [errno 98] address already in use flask)  

    $ ps -fA | grep python  
    $ kill -9 pid  
    $ flask run  
pid : pid number by naming of your flask number  


<!--stackedit_data:
eyJoaXN0b3J5IjpbLTc3NzMyNzAzMV19
-->